ivadomed/testing.py

```diff
@@ -47,17 +47,17 @@ def test(model_params, dataset_test, testing_params, path_output, device, cuda_a
                              num_workers=0)
 
     # LOAD TRAIN MODEL
-    fname_model = os.path.join(path_output, "best_model.pt")
+    fname_model = Path(path_output, "best_model.pt")
     logger.info('Loading model: {}'.format(fname_model))
     model = torch.load(fname_model, map_location=device)
     if cuda_available:
         model.cuda()
     model.eval()
 
     # CREATE OUTPUT FOLDER
-    path_3Dpred = os.path.join(path_output, 'pred_masks')
-    if not os.path.isdir(path_3Dpred):
-        os.makedirs(path_3Dpred)
+    path_3Dpred = Path(path_output, 'pred_masks')
+    if not path_3Dpred.is_dir():
+        path_3Dpred.mkdir(parents=True)
```

Comment:
This `str` to "path" conversion would require subsequent changes to ensure downstream are not affected. 

Especially the `run_inference(test_loader, model, model_params, testing_params, path_3Dpred, cuda_available, i_monteCarlo, postprocessing)` and `imed_uncertainty.run_uncertainty(ifolder=path_3Dpred)`
for Line 76 and 83 in reference to path_3Dpred